traceapp/tmpl/data_rootdir.go

```diff
@@ -0,0 +1,29 @@
+// +build dev
+
+package tmpl
+
+import (
+	"go/build"
+	"log"
+	"os"
+	"path/filepath"
+)
+
+// rootDir is the directory containing the html/template template files.
+var rootDir = filepath.Join(defaultBase("sourcegraph.com/sourcegraph/appdash/traceapp"), "tmpl")
+
+func defaultBase(path string) string {
```

Comment:
Which directory does `defaultBase` try to find?

It looks to me like it'll find the vendored copy if running via godep, and the original copy if running otherwise. Is that what it's supposed to do?

Is this original code that @beyang wrote for go-bindata's dev mode, or is it modified?